data/mods/Aftershock/mobs/zombies.json

```
@@ -518,5 +518,164 @@
       "FILTHY",
       "GUILT"
     ]
+  },
+  {
+    "id": "mon_moxie_elder",
+    "type": "MONSTER",
+    "name": { "str": "MoxPhore Elder" },
+    "description": "An arachnid humanoid stands here, Its claw-like limbs wrapped its body and head buried deep within its chest as it surveys the area around it.",
+    "default_faction": "moxie",
+    "bodytype": "human",
+    "species": [ "MOXIE" ],
+    "diff": 10,
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 270,
+    "speed": 120,
+    "material": [ "flesh" ],
+    "symbol": "E",
+    "color": "i_red",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_cut": 4,
+    "dodge": 4,
+    "vision_day": 10,
+    "regenerates": 15,
+    "armor_bullet": 30,
+    "armor_bash": 15,
+    "armor_cut": 20,
+    "harvest": "alien_moxphore",
+    "special_attacks": [
+      [ "LONGSWIPE", 20 ],
+      [ "GRAB", 20 ],
+      {
+        "id": "scratch",
+        "cooldown": 10,
+        "accuracy": 6,
+        "damage_max_instance": [
+          { "damage_type": "stab", "amount": 8, "armor_multiplier": 0.3 },
+          { "damage_type": "bash", "amount": 16, "armor_multiplier": 0.3 }
+        ]
+      }
+    ],
+    "death_function": [ "NORMAL" ],
+    "flags": [
+      "SEES",
+      "HEARS",
+      "GOODHEARING",
+      "KEENNOSE",
+      "SWIMS",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "DESTROYS",
+      "POISON",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "PUSH_VEH",
+      "FILTHY"
+    ]
+  },
+  {
+    "id": "mon_moxie_bloodhound",
+    "type": "MONSTER",
+    "name": { "str": "MoxPhore Bloodhound" },
+    "description": "An arachnid humanoid shape, twisted into a canine-like form.  It scuttles across the ground, looking blankly with vestigal eyes as it sniffs the air.",
```

```suggestion
    "description": "The remains of a long lost colonist, shaped and rebuilt into a twisted reflection of the insectoid and canid forms.  It scuttles across the ground, looking blankly with vestigal eyes as it sniffs the air.",
```
Original description is a bit weird since it describes three vastly different shapes.